### DISCUSSION TITLE
chia farm challenges -l 100 gives double outputs

### DISCUSSION THREAD

**Discussion by kiss81** [Chia-Network/chia-blockchain#4566] [Support]
Not sure if my farming is working correct now or if this is a bug as I didn't find any proof since some time now. (can be bad luck)
Plots seem to pass the filters, but when I give this command I see something abnormal as I see double challenges.

chia farm challenges -l 100

Hash: 0x8e9a02ee10895de8dcd23bb43632978b99727cxd201b2eb12bececdcf08c6cf1Index: 22
Hash: 0x8e9a02ee10895de8dcd23bb43632978b99727cxd201b2eb12bececdcf08c6cf1Index: 21
Hash: 0x8e9a02ee10895de8dcd23bb43632978b99727cxd201b2eb12bececdcf08c6cf1Index: 21
Hash: 0x8e9a02ee10895de8dcd23bb43632978b99727cxd201b2eb12bececdcf08c6cf1Index: 20
Hash: 0x8e9a02ee10895de8dcd23bb43632978b99727cxd201b2eb12bececdcf08c6cf1Index: 20
.....

(I changed one letter in the hash as I am not sure if this is "private info")

Version 1.15 amd64

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
